data-structures/hash-table.md

```diff
@@ -4,38 +4,47 @@
 4 hours
 
 ### Prerequisites
-JS
+JavaScript prototypal inheritance
 Runtime complexity
 
 ### Motivation
-Hash tables are one of the most frequestly used data structures. You'll use them in your code a lot, so knowing how and when to use hash tables is important. 
-Knowing how hash tables work will give you a deeper understanding of why hash tables are used and what they're good for.
-Hash tables are also often used in the solution to interview questions.
+Hash tables are one of the most frequestly used data structures. You'll use them in your code a lot, so knowing how and when to use hash tables is important.
+
+Knowing how hash tables work will give you a deeper understanding of why hash tables are used and what they're good for. Hash tables are also often used in the solution to interview questions.
 
 ### Objective
 **Students will be able to:** 
+- Learn about hashing algorithms
 - Know the runtime of hash table operations
 - Be able to identify problems where hash tables could be used
 - Be able to write code that uses hash tables to solve problems
 - Understand how hash tables are implemented and how this implementation leads to the runtime properties
 
 ### Specific Things To Teach
-?
+- A hash table is used to store key-value pairs or tuples.
+- Why is this called a hash table? Values are stored as a hash of the key.
```

Comment:
The second sentence seems a little misleading. What about "the hash of the key determines where the value is stored"?